documentContentProviders/dockerInspect.ts

```diff
@@ -0,0 +1,22 @@
+import { TextDocumentContentProvider, Uri, window, workspace } from "vscode";
+import { docker } from "../commands/utils/docker-endpoint";
+
+export const IMAGE_DOMAIN = "image";
+export const SCHEME = "docker-inspect";
+export const URI_EXTENSION = ".json";
```

Comment:
@chrisdias I'm adding the `.json` extension to the document `Uri` in order to get the right language service experience (e.g. colorization, folding), but I kind of hate it :) Is there a way to open a `Uri`-based document that specifies the language without an extension? I didn't see this capability in the `workspace.openTextDocument` method or any other API.